lib/msf/core/opt_rhost_url.rb

```diff
@@ -0,0 +1,75 @@
+# -*- coding: binary -*-
+
+module Msf
+  ###
+  #
+  # RHOST URL option.
+  #
+  ###
+  class OptRhostUrl < OptBase
+    def type
+      'rhost url'
+    end
+
+
+    def normalize(value)
+      return unless value
+
+      uri = get_uri(value)
+      option_hash = {}
+      # Blank this out since we don't know if this new value will have a `VHOST` to ensure we remove the old value
+      option_hash['VHOST'] = nil
+
+      option_hash['RHOSTS'] = uri.hostname
+      option_hash['RPORT'] = uri.port
+      option_hash['SSL'] = %w{ssl https}.include?(uri.scheme)
```

Comment:
```suggestion
      option_hash['SSL'] = %w[ssl https].include?(uri.scheme)
```

https://github.com/rubocop-hq/ruby-style-guide#percent-literal-braces

Reply:
During module hacking we discussed just running Rubocop over this file in its entirety, there's a things to lint such as a few unneeded uses of `and` etc.

Reply:
https://github.com/rapid7/metasploit-framework/pull/13766#pullrequestreview-445207149

Sounds good.